.github/.wordlist.txt

```diff
@@ -272,6 +274,7 @@ ConnectivityManagerImpl
 ConnectNetwork
 ConnectNetworkRequest
 ConnectNetworkResponse
+Connect's
```

Comment:
Could we rephrase things so that this is not needed to be a word? maybe `use the **Factory Data Provider** from nRF Connect ...` or similar